app/controllers/staff/activity_forms_controller.rb

```diff
@@ -70,105 +68,12 @@ def update
       @activity.assign_attributes(programme_status: DEFAULT_PROGRAMME_STATUS_FOR_FUNDS)
     end
```

Comment:
Should this get moved into `ActivityUpdater` too?

Reply:
Good call 👍